src/pages/Main/Slider/BottomSlideContents.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+export default function BottomSlideContents({ slideRef }) {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/slidetest2.json')
+      .then(res => res.json())
+      .then(data => setData(data));
+  }, []);
+
+  useEffect(() => {}, []);
+
+  return (
+    <MoveDiv
+      ref={el => {
+        slideRef.current = el;
+      }}
```

Comment:
정말 필요한 경우가 아니라면 callback ref는 권장되지 않습니다
```suggestion
    <MoveDiv
      ref={slideRef}
```
다음 문서 참고해주세요
https://ko.reactjs.org/docs/refs-and-the-dom.html#callback-refs

Reply:
수정했습니다. 공식문서 참고하겠습니다.